langchain/chains/flare/base.py

```diff
@@ -36,7 +36,7 @@ def generate_tokens_and_log_probs(
         *,
         run_manager: Optional[CallbackManagerForChainRun] = None,
     ) -> Tuple[Sequence[str], Sequence[float]]:
-        llm_result = self.generate([_input], run_manager=run_manager)
+        _, llm_result = self.generate([_input], run_manager=run_manager)
```

Comment:
Is this PR causing a breaking change? 

If so, it'll cause a breaking change for any user that implemented anything by inheriting from LLMChain